WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -140,7 +141,10 @@ public SigningState AddWitness(int index, WitScript witness)
 
 		private uint256 CalculateHash()
 			=> StrobeHasher.Create(ProtocolConstants.RoundStrobeDomain)
-				.Append(ProtocolConstants.RoundMinRegistrableAmountStrobeLabel, MinRegistrableAmount)
+				.Append(ProtocolConstants.RoundInputRegistrationTimeoutLabel, InputRegistrationTimeout)
+				.Append(ProtocolConstants.RoundConnectionConfirmationTimeoutLabel, ConnectionConfirmationTimeout)
+				.Append(ProtocolConstants.RoundOutputRegistrationTimeoutLabel, OutputRegistrationTimeout)
+				.Append(ProtocolConstants.RoundTransactionSigningTimeoutLabel, TransactionSigningTimeout)
```

Comment:
IMO this is correct, but we need to methodically work through the consequences of exactly which parameters are included and thereby effectively made static per round (as I related in https://github.com/zkSNACKs/WalletWasabi/pull/5920#issuecomment-863656183)

Reply:
They cannot change for a round because I made the `RoundState`s Id and that means that in case the coordinator changes any of those values the client can detect it (the id comparison is not implemented yet but it is trivial, we need to check the computed id of the received roundstate is exactly the same that we requested). 

What I was commenting in https://github.com/zkSNACKs/WalletWasabi/pull/5920#issuecomment-863656183 is that the coordinator's admin can configure the coordinator in a way that is creates rounds that make the graph constrain to fail anyway.